### ISSUE TITLE
Release v0.16.6

### ISSUE THREAD

**Issue by ArangoGutierrez** [kubernetes-sigs/node-feature-discovery#1930]
## Release Checklist

- [ ] Verify that the changelog in this issue is up-to-date
- [ ] Run `hack/prepare-release.sh $VERSION` to turn references to point to the upcoming release
      (README, deployment templates, docs configuration, test/e2e flags), submit a PR against the release branch
- An OWNER prepares a draft release
  - [ ] Create a draft release at [Github releases page](https://github.com/kubernetes-sigs/node-feature-discovery/releases)
  - [ ] Write the change log into the draft release
  - [ ] Upload release artefacts generated by `prepare-release.sh` script above to the draft release
- [ ] An OWNER runs
     `git tag -s -m "NFD release $VERSION" $VERSION`
- [ ] An OWNER pushes the tag with
      `git push $VERSION`
  - Triggers prow to build and publish a staging container image
      `gcr.io/k8s-staging-nfd/node-feature-discovery:$VERSION`
  - Triggers build of the documentation and publish it at
        https://kubernetes-sigs.github.io/node-feature-discovery/0.$MAJ/
- [ ] Submit a PR against [k8s.io](https://github.com/kubernetes/k8s.io), updating `registry.k8s.io/images/k8s-staging-nfd/images.yaml` to promote the container images (both "full" and "minimal" variants) to production
- [ ] Wait for the PR to be merged and verify that the image (`registry.k8s.io/nfd/node-feature-discovery:$VERSION`) is available.
- [ ] Publish the draft release prepared at the [Github releases page](https://github.com/kubernetes-sigs/node-feature-discovery/releases)
      which will also trigger a Helm repo index update to add the latest release
- [ ] Add a link to the tagged release in this issue.
- [ ] For a point release of the latest newest release branch, update README in master branch
  - [ ] Update references e.g. by running `hack/prepare-release.sh $VERSION` but **only** committing README.md, and,
        submit a PR
  - [ ] Wait for the PR to be merged
- [ ] Close this issue

**Comment by ArangoGutierrez** [comment]
I propose:

- https://github.com/kubernetes-sigs/node-feature-discovery/pull/1921
- https://github.com/kubernetes-sigs/node-feature-discovery/pull/1924

**Comment by ArangoGutierrez** [comment]
what else has gone into the v0.16 branch since v0.16.5? 
@marquiz ?

**Comment by elezar** [comment]
The following shows some github tooling changes: https://github.com/kubernetes-sigs/node-feature-discovery/compare/v0.16.5...release-0.16

**Comment by ArangoGutierrez** [comment]
Yeah, those look ok, no functional changes. that's acceptable for a point patch release

**Comment by ArangoGutierrez** [comment]
- https://github.com/kubernetes-sigs/node-feature-discovery/pull/1933

**Comment by ArangoGutierrez** [comment]
- https://github.com/kubernetes/k8s.io/pull/7470

**Comment by ArangoGutierrez** [comment]
https://github.com/kubernetes-sigs/node-feature-discovery/releases/tag/v0.16.6
